felix/ipsets/ipsets.go

```diff
@@ -681,9 +698,13 @@ func (s *IPSets) resyncIPSet(ipSetName string) error {
 		return scanner.Err()
 	})
 	if err != nil {
-		return err
+		meta.ListFailed = true
```

Comment:
```suggestion
		// This can occur if we have version skew with the version of IP set
		// used to create the IP set. Mark the metadata as invalid in order 
		// to trigger the IP set to be recreated.
		meta.ListFailed = true
```